### PR TITLE
Fixes issue #920. The splash screen doesn't stretch anymore

### PR DESCRIPTION
**Pull request by devanshi7799**
Fixes #I920

Please Add Screenshots If there are any UI changes.
ezgif com-video-to-gif (3)

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Review by devansh-299 — CHANGES_REQUESTED** [review]
Just adding the drawable file won't work . I would suggest you to refer the below link or any similar solution you can find and make the changes accordingly 
https://android.jlelse.eu/the-complete-android-splash-screen-guide-c7db82bce565

**Comment by laxyapahuja** [comment]
@devanshi7799 It seems like you've done the changes for #676 in this branch only. I'd suggest you to revert back to the last commit and do changes in a different branch.

**Comment by devanshi7799** [comment]
@laxyapahuja yaa sorry, I mixed up two different issues :/

File: app/src/main/res/layout/fragment_home_old.xml [hunk 1]
```diff
@@ -448,7 +448,7 @@
                         android:layout_marginTop="@dimen/divider_margin"
                         android:layout_width="match_parent"
                         android:text="@string/help_line_number"
-                        android:textAppearance="@style/Base.TextAppearance.AppCompat.Medium"
+                        android:textAppearance="@style/Base.TextAppearance.AppCompat.Small"
```
Comment (devansh-299) [inline app/src/main/res/layout/fragment_home_old.xml]: This change is not related to this PR, kindly undothis change

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -390,7 +390,7 @@
     <string name="select_region_qr">Select region with QR code</string>
     <!--Customer care details-->
     <string name="need_help">Contact Us</string>
-    <string name="help_line_number" translatable="false">8000000000</string>
+    <string name="help_line_number" translatable="false">+18000000000</string>
```
Comment (devansh-299) [inline app/src/main/res/values/strings.xml]: Please don't change the helpline number

**Comment by devansh-299** [comment]
Please do the suggested changes and squash all your commits

**Comment by ShivangiSingh17** [comment]
@devanshi7799 can you please squash all your commits.

**Review by ShivangiSingh17 — COMMENTED** [review]
The changes made aren't giving the expected results.

**Pull request closed without merging**